### PR TITLE
Upgraded the meshkit utility runner for the mesheryctl error codes

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Upgraded the meshkit utility runner for the mesheryctl error codes (That was commented previously)

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ADAPTER_URLS := "mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001 mesherylocal.layer5.io:10002 mesherylocal.layer5.io:10003 mesherylocal.layer5.io:10004 mesherylocal.layer5.io:10006 mesherylocal.layer5.io:10007 mesherylocal.layer5.io:10008 mesherylocal.layer5.io:10009"
-
+
```
Comment (leecalcote) [inline Makefile:2]:
```suggestion

```

File: .github/workflows/error-codes-updater.yaml [hunk 2]
```diff
@@ -46,5 +45,5 @@ jobs:
           commit_user_name: l5io
           commit_user_email: ci@layer5.io
           commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          commit_options: '--signoff'
+          commit_options: "--signoff"
```
Comment (leecalcote) [inline .github/workflows/error-codes-updater.yaml:48]:
```suggestion
          commit_options: "--signoff" 
```

File: .github/workflows/error-codes-updater.yaml [hunk 3]
```diff
@@ -46,5 +45,5 @@ jobs:
...
```
Comment (leecalcote) [inline .github/workflows/error-codes-updater.yaml:48]:
```suggestion
          commit_options: "--signoff"
```

**Review by DelusionalOptimist — CHANGES_REQUESTED** [review]
Going good. Just some minor changes.
Also, create a new target named `error` in mesheryctl's Makefile similar to the one we have in the one for meshery server.

**Comment by DelusionalOptimist** [comment]
Aah its becoming meshy here XD

**Comment by sayantan1413** [comment]
So @DelusionalOptimist should I make a new PR for this. It makes a bit of meshy right😄

File: .github/workflows/error-codes-updater.yaml [hunk 4]
```diff
@@ -2,9 +2,9 @@ name: Meshkit Error Codes Utility Runner
 on:
   push:
     branches:
-      - 'master'
```
[the remark this answers is not in the document]
Comment (sayantan1413) [reply .github/workflows/error-codes-updater.yaml:5]: Yup I will change it.

**Comment by DelusionalOptimist** [comment]
No worries. I think we can wait until the mesh gets cleared. I'll help you restore sanity on this later : )

**Pull request closed without merging**